File: scripts/audit/fixtures/PAN003-locale-parity/bad/scripts/lib/locales.py
"""Fixture (PAN003-locale-parity / bad): Python locale SoT that DISAGREES with
the sibling src/lib/locales.ts (only "ru" here, but TS lists "ru","en"). The
audit execs this module, so it must import."""

LOCALES: tuple[str, ...] = ("ru",)
DEFAULT_LOCALE: str = "ru"

File: audit/fixtures/PAN003-locale-parity/bad/src/lib/locales.ts
// Fixture (PAN003-locale-parity / bad): TS locale SoT that DISAGREES with the
// sibling pancratius/locales.py (TS has two locales, Python has one) — the
// cross-language audit must fire.
export const LOCALES = ["ru", "en"] as const;
export type Locale = (typeof LOCALES)[number];
export const DEFAULT_LOCALE: Locale = "ru";
